packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-services.4/VisualRegressionService.tool.mdx

```diff
@@ -4,7 +4,7 @@ import { Notice } from '@cerner/terra-docs';
 
 Visual regression testing for WebdriverIO.
 
-Based on the work of Jan-André Zinser on [wdio-visual-regression-service](https://github.com/zinserjan/wdio-visual-regression-service) and [wdio-screenshot](https://github.com/zinserjan/wdio-screenshot). The visual regression service performs image comparison evaluations to determine mismatch tolerances. Screenshots exceeding the mismatch tolerance will fail the test run.
+Based on the work of Jan-André Zinser on [wdio-visual-regression-service](https://github.com/zinserjan/wdio-visual-regression-service) and [wdio-screenshot](https://github.com/zinserjan/wdio-screenshot). The visual regression service performs image comparison evaluations to determine mismatch tolerances. Screenshots exceeding the mismatch tolerance will fail the test run. Screenshot failure due to mismatch tolerance can be disabled with the `--ignoreScreenshotMismatch` option.
```

Comment:
nit: Should we clarify that consumers generally should not use --ignoreScreenshotMismatch just to get a test run to pass. The intention of this option is for a dry run or for CI so consumers don't abuse this option.

Reply:
As per offline discussion, reworded the sentence a here https://github.com/cerner/terra-toolkit/pull/655/commits/7b22a7c62d5b4a236bb47def840c973e0e08289b